cmd/markdown/support.md

```
@@ -0,0 +1,29 @@
+
+[🔗](https://github.com/sponsors/cloudposse)
+
+Included with sponsorship, we host **30-minute workshops twice a week** to:  
+- Answer your questions.  
+- Assist with debugging issues.  
+- Discuss architectural decisions.  
+
+Workshops are tailored to help you succeed with Atmos, our AWS Reference Architectures, and related tools.  
+
+> **Note**: Paid support includes priority responses.
+>           [](https://github.com/sponsors/cloudposse)
```

Markdownlint (0.37.0)

Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)